### ISSUE TITLE
Add "Live Transcribe" to Implementation Status

### ISSUE THREAD

**Issue by Rexadev** [microg/GmsCore#1890]
https://github.com/microg/GmsCore/wiki/Implementation-Status

https://www.android.com/accessibility/live-transcribe/
https://www.youtube.com/watch?v=jLCwjIaPXwA

**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
Not Applicable

**Describe alternatives you've considered**
This App by Google - https://play.google.com/store/apps/details?id=com.google.audio.hearing.visualization.accessibility.scribe

**Additional context**
Not Applicable

**Comment by ale5000-git** [comment]
The app works for me, there are erros in the logcat but in general it works.

**Comment by megvadulthangya** [comment]
To me doesn't detect my voice, no matter if i speak engrish or Hungarian

**Comment by ale5000-git** [comment]
It detect the voice on mine (in Italian).
I have installed microG with `microG unofficial installer` and downloaded the last version of Live Transcribe from apkmirror.

**Comment by megvadulthangya** [comment]
thank you Ale! You installed the complete version? I installed some pico or minimal version of microg dont even remember....

**Comment by ale5000-git** [comment]
There isn't any pico or minimal version, it is the one created by me: https://github.com/micro5k/microg-unofficial-installer

**Comment by megvadulthangya** [comment]
Yup! I installed original the MinMicroG-Standard-2.11.1-20220202153527.zip ... I removed with Titanium backup then installed your thing: microg-unofficial-installer-v1.3.0-beta-by-ale5000-signed.zip

Installed the app from Aurora Store, same thing happend... If i talk i see its detecting, on the right top corner but dont wrote anything, ENG or HUN... But if i snap my finger or whistle, wrote finger snapping or bird whistle... I go to apkmirror, but i expect the same result

**Comment by ale5000-git** [comment]
I'm not sure, it works on mine.
Maybe try posting a logcat.

**Comment by megvadulthangya** [comment]
Im not the best with logcat...  My phone has CrDroid, deafultly comes with MatLog I saved [new logs, ](https://cl0ud.sytes.net/s/8sN8ozS7LfFyMNe)then go into Live Transcribe talk then copy the logs, i hope you can understand what is have in those log file...

**Comment by ale5000-git** [comment]
The simple way is:

1) First:
`adb logcat -c`
it clear previous logcat messages.

2) Then leave this open:
`adb logcat -v long > debug.log`

3) Then on the phone open and use the app a lot

4) Wait some time and the end close the thing at point 2 that should have been stil active.

5) Now you have a `debug.log` file.

**Comment by megvadulthangya** [comment]
Thank you for your teachings! 

I watch a 8 aprox minute english video snapping finger whistling while the app is opened... Changed the mic from Auto detect to The Device's mic... wait some time then change back to auto detect... then ctrl+c the powershell to stop logging... And there is the created [debug.log](https://cl0ud.sytes.net/s/e3NzJ2XmewB946W)
